go/src/koding/klientctl/metrics/bolt.go

```diff
@@ -0,0 +1,89 @@
+package metrics
+
+import (
+	"io"
+	"strconv"
+	"time"
+
+	"github.com/boltdb/bolt"
+)
+
+var _ io.WriteCloser = &BoltConn{}
+
+// BoltConn writes and reads to BoltDB
+type BoltConn struct {
```

Comment:
naming nit: Can we name this BoltQueue? When I see `BoltConn` I don't know what kind of functionality expect without going through the methods, I think `BoltQueue` gives a hint.